client/src/components-questions/photos/aphotos.jsx

```diff
@@ -4,7 +4,7 @@ import Aphoto from './aphoto.jsx';
 
 const Aphotos = ({photos}) =>{
   return (
-    <div>
+    <div role="photos">
```

Comment:
Great for using a role! but It'd be best to use an accessible role https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles

Maybe "feed" or "presentation" I'm not sure of the use case